src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss

```diff
@@ -0,0 +1,8 @@
+@import '../../../../globals/vars';
+
+.#{$iot-prefix}--btn--icononly {
+  margin-left: 0 !important; // !important needed to override the styled component value
```

Comment:
Could we instead increase the scope here to attempt to migrate off of styled-components entirely for the `RowActionsCell` component?

Reply:
I kept it to a minimum on this PR and opened https://github.com/IBM/carbon-addons-iot-react/issues/1005 to do the refactoring

Reply:
importants are bad usually, can we increase the specificity in a different way to force the override?

Reply:
@scottdickerson @tay1orjones thanks! i updated a bit the css to avoid !important (feels better now)